### PR TITLE
Updates to schedule

### PR DESCRIPTION
**Pull request by Albadylic**
Jack and Gregor to review on Monday

### CODE REVIEW

File: projects/tech-for-better/build-1.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 | Time    | Activity                     |
 | ------- | ---------------------------- |
 | 09:45   | Check-in                     |
-| 10:00   | Reach out to Project Mentors & Role mentors |
+| 10:00 | Feedback Survey |
+| 10:10   | Reach out to Project Mentors & Role mentors |
```
Comment (jackherizsmith) [inline projects/tech-for-better/build-1.md:9]: I wasn't sure about this in SP build 2 so removed it - I'm not sure if we want to allocate a specific time or leave it to them to find each other as and when they need it?

**Review by jackherizsmith — COMMENTED** [review]
We might need to put a bit  more thought into tech spikes - unless instead we said Thurs pm could be spent doing deployment? Get all the boring stuff out of the way ahead of build week 1?

File: projects/tech-for-better/build-1.md (hunk 1 above)
Comment (Albadylic) [reply projects/tech-for-better/build-1.md:9]: Happy to remove

**Comment by Albadylic** [comment]
> We might need to put a bit more thought into tech spikes - unless instead we said Thurs pm could be spent doing deployment? Get all the boring stuff out of the way ahead of build week 1?

Yes, we could call it project setup. I'd be curious to try Tech Spikes again with little direction and see how they go.

**Comment by jackherizsmith** [comment]
I can't ask you to review your own PR apparently but I have made some more changes

**Comment by Albadylic** [comment]
All good - cheers!

**Pull request merged**